packages/ui/src/components/AccountSelection.tsx

```diff
@@ -1,104 +1,133 @@
-import { Autocomplete, Box, InputAdornment, TextField } from "@mui/material";
-import Identicon from "@polkadot/react-identicon";
-import { ChangeEvent, useCallback, useEffect, useMemo, useRef, useState, SyntheticEvent } from "react";
-import { styled }  from "@mui/material/styles";
-import { useAccounts } from "../contexts/AccountsContext";
-import { InjectedAccountWithMeta } from "@polkadot/extension-inject/types"
+import { Autocomplete, Box, InputAdornment, TextField } from '@mui/material';
+import Identicon from '@polkadot/react-identicon';
+import {
+  ChangeEvent,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+  SyntheticEvent,
+} from 'react';
+import { styled } from '@mui/material/styles';
+import { useAccounts } from '../contexts/AccountsContext';
+import { InjectedAccountWithMeta } from '@polkadot/extension-inject/types';
 import { createFilterOptions } from '@mui/material/Autocomplete';
-import { isValidAddress } from "../utils";
-import { ICON_THEME, ICON_SIZE } from "../constants";
-import { useAccountNames } from "../contexts/AccountNamesContext"
+import { isValidAddress } from '../utils';
+import { ICON_THEME, ICON_SIZE } from '../constants';
+import { useAccountNames } from '../contexts/AccountNamesContext';
 import IconButton from '@mui/material/IconButton';
 import AddIcon from '@mui/icons-material/Add';
 
 interface Props {
   className?: string;
-  addressDisabled?: boolean
-  nameDisabled?: boolean
-  addSignatory?: (address: string) => void
-  value?: string
-  inputLabel?: string
-  currentSignatories?: string[]
-  withName?: boolean
-  withAddButton?: boolean
+  addressDisabled?: boolean;
+  nameDisabled?: boolean;
+  addSignatory?: (address: string) => void;
+  value?: string;
+  inputLabel?: string;
+  currentSignatories?: string[];
+  withName?: boolean;
+  withAddButton?: boolean;
 }
 
 const filterOptions = createFilterOptions({
   ignoreCase: true,
-  stringify: (option: InjectedAccountWithMeta) => option.address + option.meta.name,
+  stringify: (option: InjectedAccountWithMeta) =>
+    option.address + option.meta.name,
 });
 
 const getOptionLabel = (option: string | InjectedAccountWithMeta | null) => {
-  if (!option) return ""
-
-  return typeof option === "string"
-    ? option
-    : option.address
-}
-const AccountSelection = ({ className, addSignatory, addressDisabled = false, nameDisabled = false, value, inputLabel = "Account", currentSignatories = [], withName = false, withAddButton = false }: Props) => {
-  const { accountList = [], getAccountByAddress } = useAccounts()
-  const [selected, setSelected] = useState(value)
-  const [errorMessage, setErrorMessage] = useState("")
-  const ref = useRef<HTMLInputElement>(null)
-  const { accountNames, addName } = useAccountNames()
-  const [name, setName] = useState("")
+  if (!option) return '';
+
+  return typeof option === 'string' ? option : option.address;
+};
+const AccountSelection = ({
+  className,
```

Comment:
The function params is a place where I'd like to keep things on one line. This is read very rarely, and bloating the line number doesn't help readability.